src/main/java/com/meraphorce/services/UserService.java

```diff
@@ -20,4 +21,22 @@ public User createUser(User user){
     public  List<User> getAllUsers(){
         return userRepository.findAll();
     }
+
+    public User getUserById(String id){
+        return userRepository.getById(id);
+    }
+
+    public EmptyResponse deleteUser(String id){
```

Comment:
Instead of using EmptyResponse, use the response from the controller 
ResponseEntity<Void> and this method in the service layer make it as void